liblangutil/SourceLocation.cpp

```diff
@@ -27,6 +27,34 @@ using namespace solidity;
 using namespace solidity::langutil;
 using namespace std;
 
+string SourceLocation::singleLineSnippet(StringMap const& _sourceCodes) const
```

Comment:
I don't fully remember, but I think I specifically took the functionality to extract the source snippet out of this class, and I think it is nicer if this class just stays a data container without extra functionality, but at least we should find this place and unify it if possible.

Reply:
I could make it a standalone function in that file. Or move it to `liblangutil/Common.h`.

Reply:
It's `CharStream::text`

Reply:
Ah, that's what you meant. Right. I can add it next to that helper.

Reply:
Moved to `CharStream`.